src/diffusers/models/attention.py

```diff
@@ -154,7 +154,7 @@ def __init__(
         self.ff = FeedForward(dim, dropout=dropout, activation_fn=activation_fn, final_dropout=final_dropout)
 
         # 4. Fuser
-        if attention_type == "gated":
+        if attention_type == "gated" or attention_type == "GatedTextImage":
```

Comment:
"gated_text_image" is a better name here IMO.

Reply:
```suggestion
        if attention_type == "gated" or attention_type == "gated-text-image":
```
Can we stick to lowercase here? :-)